### PR TITLE
Fix the focal Tversky loss

### PR DESCRIPTION
**Pull request by zifuwanggg**
Fix the focal Tversky loss according to Eq (4). of the [paper](https://arxiv.org/pdf/1810.07842.pdf). The gamma term is applied per-class.

### CODE REVIEW

**Comment by qubvel** [comment]
Probably, the trailing space should be removed in the end of line

**Comment by zifuwanggg** [comment]
There seems to be a new line in the master branch, but I have removed it in the most recent commit.

**Comment by zifuwanggg** [comment]
@qubvel Commit [5410122](https://github.com/qubvel/segmentation_models.pytorch/pull/737/commits/54101221a61d3d20e9e5be252dee34e60ab4e99a) fails the code formatting check.

The only change I've made, comparing with the original file, is to change `loss.mean() ** self.gamma` to `(loss ** self.gamma).mean()`. The error message from `Black` is not very helpful. Could you please have a look?

File: segmentation_models_pytorch/losses/tversky.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def __init__(
         self.gamma = gamma
 
     def aggregate_loss(self, loss):
-        return loss.mean() ** self.gamma
+        return (loss ** self.gamma).mean()
```
Comment (mukhery) [inline segmentation_models_pytorch/losses/tversky.py:56]: @zifuwanggg to fix black
```suggestion
        return (loss**self.gamma).mean()
```

**Pull request closed without merging**